src/com/apmath/loans/domain/models/loans/Loan.kt

```diff
@@ -32,6 +33,7 @@ class Loan(
     override var completed: Boolean = false
         private set
 
+    @Transient
```

Comment:
We are using serialization? What are the purposes?

Reply:
In [specification](https://docs.google.com/document/d/1RiURQhXa6QD7rrq4720uTNWcksZWoBAFmj54RSLcgUg/edit#bookmark=id.cpdcdy6gb1mp), we respond Loans without payments

Reply:
It is domain model, when you want to return Loan you should provide application loan model and map domain loan into application loan

Reply:
Look in calculations

Reply:
@malinink is it ok now?

Reply:
@BarracudaPff looks good